ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs

```diff
@@ -51,6 +99,17 @@ public async Task<IActionResult> OnPostAsync(long urn)
 			{
 				CaseUrn = (CaseUrn)urn;
 
+				if (!ModelState.IsValid)
+				{
+					TempData["Decision.Message"] = string.Join(",\r\n", ModelState.Values.SelectMany(v => v.Errors).Select(e => e.ErrorMessage).ToArray());

```

Comment:
I might've got this wrong, I think you mentioned that the front end would take care of it and so this shouldn't concatenate the errors ?